Mathlib/Init/Data/Nat/Basic.lean

```diff
@@ -7,6 +7,12 @@ import Mathlib.Init.ZeroOne
 import Mathlib.Init.Data.Nat.Notation
 import Mathlib.Util.CompileInductive
 
+/-!
```

Comment:
I have a similar comment to some other files: this could be a good start if this file had 200 lines.
Given that it very short (and contains only deprecated lemmas), I wonder if adding this doc comment is the best thing to do. It could be better to try to remove these lemmas instead. (There was a zulip thread recently about removing bit0/1 lemmas; see also #11414; this is slightly more involved.)

Reply:
I am going to remove this file soon (as well as anything under `Mathlib.Init`), so indeed there is little point in writing this module doc

Reply:
Great, good to know.